cmd/root.go

```diff
@@ -0,0 +1,59 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"github.com/wish/gatekeeper/verifier"
```

Comment:
wrong ordering of imports.

goimports -local github.com/wish/gatekeeper

https://github.com/golang/go/wiki/CodeReviewComments#imports
https://stackoverflow.com/questions/38704769/golang-import-grouping-by-package